pkg/authorization/cache/policy_cache.go

```diff
@@ -18,6 +18,18 @@ import (
 	bindingregistry "github.com/openshift/origin/pkg/authorization/registry/policybinding"
 )
 
+// PolicyCacheAdapter exposes getter methods for policy indexers and reflectors for the project cache
+type PolicyCacheAdapter interface {
+	// Return refrences to the indexers for the project cache to use
+	GetPolicyBindingIndexer() *cache.Indexer
+	GetPolicyIndexer() *cache.Indexer
+	GetClusterPolicyBindingIndexer() *cache.Indexer
+	GetClusterPolicyIndexer() *cache.Indexer
+
+	// Returns reflectors backing this cache for the project cache to use in determining synchronization skips
+	GetBackingReflectors() []*cache.Reflector
```

Comment:
having a bucket of reflectors doesn't look right.  Why aren't these individual to let the callers decide what they want?